src/Concerns/CapturesState.php

```diff
@@ -673,6 +679,11 @@ public function shouldCaptureLogs(): bool
         return $this->enabled();
     }
 
+    public function sampleScheduledTask(Event $event, float $rate = 1.0): void
+    {
+        $this->scheduledTasksSampleRates[$event] = $rate;
+    }
+
```

Comment:
I think this should probably be an internal method only called from our mixin to mark a schedule task sample rate. Any thoughts